open-machine-learning-jupyter-book/assignments/data-science/matplotlib-applied.ipynb

```diff
@@ -33,8 +33,7 @@
     "* Scatter Plots, subplots\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #13.            actual_result = label_encode(test_df, "c1", "c1")
</pre><p>give a different name, e.g. encoded_c1, for the new column name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>